collector.go

```diff
@@ -101,15 +101,15 @@ func (c Collector) measure() error {
 
 		log.Printf("WARN: clock drift is above %.2fs, taking multiple measurements for %.2f seconds", highDrift, c.NtpMeasurementDuration.Seconds())
 		for time.Since(begin) < c.NtpMeasurementDuration {
-			clockOffset, stratum, err := c.getClockOffsetAndStratum()
+			var stratum float64
+			clockOffset, stratum, err = c.getClockOffsetAndStratum()
```

Comment:
What's the rationale here? Why is `clockOffset` being reused, but `strat` vs. `stratum` gets a new variable?

In general, I prefer variables being limited in their scope like in the original version.

Reply:
`stratum` gets a new variable because this is where it is first being used hence the declaration. It can't be reused like `clockOffset`.

For what it's worth, I also prefer variables being limited to scope but I like having `go vet` complain about shadowing because in some cases it makes sense like in one of Limes' `Scrape` methods, `client` name was reused multiple times but it was Nova client in some places and Placement client in others so I renamed the Nova one to `novaClient`.

Can we rename here like `innerClockOffset` (just an example, I don't know what's suitable here) or should I revert to original and add a `nolint` here?

Reply:
I have pushed a commit to clarify the variable naming.